p5js/01_ImageNet_Camera/sketch.js

```diff
@@ -0,0 +1,28 @@
+/* ===
+ML5 Example
+01_ImageNet_Camera
+Video Camera Classification using p5.js
+=== */
+
+// Initialize the ImageNet method with the MobileNet model.
+const classifier = new ml5.ImageNet('MobileNet');
+let video;
+
+function setup() {
+  noCanvas();
+  video = createCapture(VIDEO);
+  // Start the prediction loop!
+  guess();
```

Comment:
This probably fails the first time since the video isn't ready but I guess it's ok? It's simple and it works!

Reply:
Updated this so now `guess()` is called as a callback of `createCapture()`